CONTRIBUTING.md

```diff
@@ -1,3 +1,23 @@
 # Contributing
 
-More to come soon...
+All contributions are welcome to this project!
+
+The general process for contributing code is as follows.
+
+1. Open and issue describing the desired feature, bug fix, or improvment you'd like to see before making any changes.
```

Comment:
`Open an issue describing the desired feature, bug fix, or improvement you'd like to see before making any changes.`